packages/query/src/block-processor.ts

```diff
@@ -318,28 +305,23 @@ export class BlockProcessor implements BlockProcessorInterface {
       // this is a network query
       const blockTx = await this.querier.app.txsByHeight(compactBlock.height);
 
-      // identify tx that involve a new record
-      // - compare nullifiers
-      // - compare state commitments
-      // - collect relevant tx for info generation later
-      // - if matched by commitment, collect record with recovered source
```

Comment:
reason for removing code comment?

Reply:
Carried over to external function. But will add a code comment above where its used again.